Sources/FluentKit/Query/QueryBuilder+Unique.swift

```diff
@@ -0,0 +1,15 @@
+extension QueryBuilder {
+    public func all<Result>(selecting key: KeyPath<Model, Field<Result>>) -> EventLoopFuture<[Result]> where Result: Decodable {
```

Comment:
The term "select" isn't really used elsewhere in Fluent. That's more of a SQL thing. I think `all(\.$name)` is perfectly readable honestly. Especially since we have `count(\.$name)`.

Reply:
Will update :+1: